substrate/frame/bags-list/src/lib.rs

```diff
@@ -392,6 +454,18 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		Ok(maybe_movement)
 	}
 
+	/// Removes an account from the list if their score as per the score provider is zero.
+	///
+	/// Returns whether the account was removed.
+	pub fn remove_if_zero(account: &T::AccountId) -> Result<bool, DispatchError> {
```

Comment:
In which scenario can this happen? Shouldn't they be removed when they become zero from a non-zero score?